### ISSUE TITLE
Allow likes/appreciation on top of stars

### ISSUE THREAD

**Issue by minitauros** [isaacs/github#1242]
When I star a repository it's because I use it and I want to remember it and I perhaps want to keep track of it and be able to easily find it back.

Sometimes I come across a repository that I really do like, but that I don't use, or cannot use because it is in a language that I am not using. I would nonetheless like to show my appreciation of such repositories. I don't want to star it, because it would mess with my list of favorite repositories and it would become a search to distinguish between repositories that I like and that I use or have used.

Therefore I would like to request that it become possible to "like" (or "appreciate", or whatever term fits best here) repositories. That way it could also become clearer which repositories are being used (stars, but maybe more likes) vs which ones are state of the art (stars, but perhaps even more likes), or both.

**Comment by Larivact** [comment]
No, this just unnecessarily complicates GitHub. Just use regular browser bookmarks to track repositories you use.
